### PR TITLE
Update helpers.ts

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Ignore additional msg

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/src/utils/postMessage.ts [hunk 1]
```diff
@@ -6,14 +6,18 @@ export type MessageSender = {
   postMessage: (message: string) => void
 }
 
-const SPAMMY_MESSAGES = new Set<string>([MessageType.GET_POSITION])
+const IGNORE_MESSAGES = new Set([
+  MessageType.GET_POSITION,
+  MessageType.SUBMIT_SIGNIN,
```
[the remark this answers is not in the document]
Comment (raymondjacobson) [reply packages/mobile/src/utils/postMessage.ts:11]: Adding all sign up related actions

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/raymondjacobson-patch-1

**Review by dylanjeffers — APPROVED** [review]
cool i like this

**Comment by raymondjacobson** [comment]
> cool i like this

i don't love it. should be the opposite for redaction purposes i think but we're dropping all this code soon

**Pull request merged**